pkg/preflight/assets/host-preflights.yaml

```
@@ -140,6 +152,45 @@ spec:
               message: The disk containing directory /var/lib/docker has less than 10Gi of disk space available
           - pass:
               message: The disk containing directory /var/lib/docker has at least 30Gi of total space, has at least 10Gi of disk space available, and is less than 60% full.
+    - diskUsage:
+        checkName: "Ephemeral Disk Usage /var/lib/containerd"
+        collectorName: "Ephemeral Disk Usage /var/lib/containerd"
+        exclude: '{{kurl not .Installer.Spec.Containerd.Version}}'
+        outcomes:
+          - warn:
+              when: "used/total > 80%"
+              message: The disk containing directory /var/lib/containerd is more than 80% full
```

```suggestion
          - fail:
              when: "total < 30Gi"
              message: The disk containing directory /var/lib/docker has less than 30Gi of total space
          - fail:
              when: "used/total > 80%"
              message: The disk containing directory /var/lib/docker is more than 80% full
          - warn:
              when: "used/total > 60%"
              message: The disk containing directory /var/lib/docker is more than 60% full
```

i would suggest following the docker preflight here

https://kurl.sh/docs/install-with-kurl/system-requirements

```
containerd | /var/lib/containerd/ | 30 GB *
```